packages/trader/src/Modules/CFD/Containers/props.types.ts

```diff
@@ -1,8 +1,11 @@
+import { FormikHelpers as FormikActions } from 'formik';
```

Comment:
Why do we use alias here?

Reply:
@matin-binary According to Formik docs https://formik.org/docs/migrating-v2 , alias has to be used in v2 (i.e. Formik version 2.x.*). So since we are using Formik v2.1.4, I opted to stick to the docs for v2:
![Screenshot 2022-01-20 at 09 59 12](https://user-images.githubusercontent.com/87321516/150289660-984a20ba-1d2a-4e39-b27c-297173c64c63.png)

Reply:
Yes let's add it as `FormikHelpers`